Makefile

```diff
@@ -27,6 +27,8 @@ deploy-appbundle:
 	cp \
 		$(BUILD_DIR)/app.bundle.min.js \
 		$(BUILD_DIR)/app.bundle.min.map \
+		$(BUILD_DIR)/do_external_connect.min.js\
+		$(BUILD_DIR)/do_external_connect.min.map\
```

Comment:
No, we can't use this name. `external_connect` is optional js file that provides connection optimization. Some Jitsi deployments inject `external_connect.js` here and `do_external_connect` uses it to create connection before lib-jitsi-meet and jitsi-meet will be loaded. I think we need this change because now we bundle `do_external_connect` via webpack and it's better to have all bundles in one directory.

We're using webpack for bundling because functions such as `getConfigParamsFromUrl` are used in `do_external_connect` and `app.bundle.js` and we can't concatenate these to files because the key idea of it is to load `do_external_connect` before. @hristoterezov WDYT?